content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -71,7 +72,11 @@ Decide how to expose the main cluster's Istiod discovery service to
 the remote clusters. Choose between **one** of the two options below:
```

Comment:
Feel free to ignore if this out out of scope for this PR, but I would cut the word "between".

(You can say "choose between the two options" or you can say "choose one of the two options", but saying "choose between one of the two options" isn't quite grammatical.)

Reply:
Nitpicky followup thought: Also consider slight changes to the phrasing here to match the phrasing below, where it says "Pick one of the two options". So you could remove the bolding on "one" here and change "Choose" to "Pick", to be consistent with the later line. But fine to leave this alone.

Reply:
```suggestion
the remote clusters. Pick one of the two options:
```